onmt/translate/Translator.lua

```diff
@@ -28,11 +28,15 @@ end
 
 function Translator:__init(args)
   self.opt = args
-  onmt.utils.Cuda.init(self.opt)
```

Comment:
Translation servers expect the `Translator` to initialize the GPU. We should either keep this line or update the servers code accordingly.